initdb.d/littil-backup.sql

```diff
@@ -0,0 +1,473 @@
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `location` (
+  `location_id` binary(16) NOT NULL,
+  `country_code` varchar(2) NOT NULL,
+  `address` varchar(255) NOT NULL,
+  `postal_code` varchar(10) NOT NULL,
+  `latitude` mediumint(9) DEFAULT NULL,
+  `longitude` mediumint(9) DEFAULT NULL,
+  `created_by` binary(16) DEFAULT NULL,
+  `created_date` datetime DEFAULT NULL,
+  `last_modified_by` binary(16) DEFAULT NULL,
+  `last_modified_date` datetime DEFAULT NULL,
+  PRIMARY KEY (`location_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `location`
+--
+
+LOCK TABLES `location` WRITE;
+/*!40000 ALTER TABLE `location` DISABLE KEYS */;
+/*!40000 ALTER TABLE `location` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `school`
+--
+
+DROP TABLE IF EXISTS `school`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `school` (
+  `school_id` binary(16) NOT NULL,
+  `school_name` varchar(255) NOT NULL,
+  `location` binary(16) NOT NULL,
+  `user` binary(16) NOT NULL,
+  `contact_person` binary(16) NOT NULL,
+  `created_by` binary(16) DEFAULT NULL,
+  `created_date` datetime DEFAULT NULL,
+  `last_modified_by` binary(16) DEFAULT NULL,
+  `last_modified_date` datetime DEFAULT NULL,
+  PRIMARY KEY (`school_id`),
+  KEY `fk_school_location` (`location`),
+  KEY `fk_school_user` (`user`),
+  KEY `fk_school_contact_person` (`contact_person`),
+  CONSTRAINT `fk_school_contact_person` FOREIGN KEY (`contact_person`) REFERENCES `contact_person` (`contact_person_id`),
+  CONSTRAINT `fk_school_location` FOREIGN KEY (`location`) REFERENCES `location` (`location_id`),
+  CONSTRAINT `fk_school_user` FOREIGN KEY (`user`) REFERENCES `user` (`user_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `school`
+--
+
+LOCK TABLES `school` WRITE;
+/*!40000 ALTER TABLE `school` DISABLE KEYS */;
+/*!40000 ALTER TABLE `school` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `user`
+--
+
+DROP TABLE IF EXISTS `user`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `user` (
+  `user_id` binary(16) NOT NULL,
+  `provider` varchar(50) NOT NULL,
+  `provider_id` varchar(255) DEFAULT NULL,
+  `email_address` varchar(255) NOT NULL,
+  `created_by` binary(16) DEFAULT NULL,
+  `created_date` datetime DEFAULT NULL,
+  `last_modified_by` binary(16) DEFAULT NULL,
+  `last_modified_date` datetime DEFAULT NULL,
+  PRIMARY KEY (`user_id`),
+  UNIQUE KEY `email_address` (`email_address`),
+  UNIQUE KEY `uc_user_email_address` (`email_address`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `user`
+--
+
+LOCK TABLES `user` WRITE;
```

Comment:
reviewed; imho safe, it does not contain credentials but does contain (functional) email addresses. This however needs attention in reviewing test-data in pull requests